plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/ScriptSelectorPanel.java

```diff
@@ -73,7 +74,9 @@
     @NotNull
     private final IFolder rootFolder;
     @NotNull
-    private final List<ResourceInfo> scriptFiles;
+    private List<ResourceInfo> scriptFiles;
+
+    private List<ResourceInfo> projectScriptFiles;
```

Comment:
```suggestion
    private final List<ResourceInfo> scriptFiles;
    private final List<ResourceInfo> projectScriptFiles;
```

And both can be initialized inside the constructor.